api/RTCRtpTransceiverInit.json

```diff
@@ -175,10 +175,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
This (`streams`) was added in https://github.com/WebKit/WebKit/commit/d74a65c152461a0c0fd1b708368eb53a6f52151a which suggests it should be supported and this shouldn't be false.

Reply:
Good catch!  I've double-checked everything set to `false`, and luckily only this and `sendEncodings` were inaccurate.  I've fixed both!

Reply:
That was a lucky random sampling I did then, it's my lucky day :)